perllib/FixMyStreet/App/Controller/Admin/Bodies.pm

```diff
@@ -272,6 +272,11 @@ sub update_contact : Private {
     } else {
         $contact->unset_extra_metadata( 'photo_required' );
     }
+    if ( $c->get_param('open311_protect') ) {
+        $contact->set_extra_metadata_if_undefined( open311_protect => 1 );
```

Comment:
Is this for the weird case where we've manually set open311_protect to 0 or something? Just think `set_extra_metadata` on its own might be more obvious (I realise it's copying photo_required above). Perhaps I've missed something.

Reply:
Pure cargo culting. Will remove `_if_undefined` :)